tasks.py

```diff
@@ -140,3 +155,142 @@ def get_config(config=None):
+        print('\n'.join(textwrap.wrap(
+            'GITHUB_TOKEN env variable is required. Set up a personal token here:\n'
+            'https://github.com/settings/tokens'
+        )))
+        return False
+
+    # Current base for labels across repos
+    labels = [
+        Label('Accepted', '10ff91', 'Accepted issue on our roadmap', []),
+        Label('Bug', 'FF666E', 'A bug', ['bug']),
+        Label('Design', 'e10c02', 'Design or UX/UI related', []),
+        Label('Feature', '5319e7', 'New feature', ['Feature Overview']),
+        Label('Good First Issue', 'bfe5bf', 'Good for new contributors', ['good first issue']),
+
+        Label('Improvement', 'e2419d', 'Minor improvement to code', ['enhancement', 'Enhancement']),
+
+        Label('Needed: design decision', '54473F', 'A core team decision is required', []),
+        Label('Needed: documentation', '54473F', 'Documentation is required', []),
+        Label('Needed: more information', '54473F', 'A reply from issue author is required', []),
+        Label('Needed: patch', '54473F', 'A pull request is required', []),
+        Label('Needed: replication', '54473F', 'Bug replication is required', []),
+        Label('Needed: tests', '54473F', 'Tests are required', []),
+
+        Label('Operations', '0052cc', 'Operations or server issue', []),
+
+        Label('PR: hotfix', 'fbca04', 'Pull request applied as hotfix to release', []),
+        Label('PR: work in progress', 'F0EDF9', 'Pull request is not ready for full review', []),
+
+        Label('Priority: high', 'e11d21', 'High priority', ['High Priority', 'Priority: High']),
+        Label('Priority: low', '4464d6', 'Low priority', ['Priority: Low']),
+
+        Label('Sprintable', 'fef2c0', 'Small enough to sprint on', []),
+
+        Label('Status: blocked', 'd0d0c0', 'Issue is blocked on another issue', []),
+        Label('Status: stale', 'd0d0c0', 'Issue will be considered inactive soon', []),
+
+        Label('Support', '5494E8', 'Support question', ['question', 'Question']),
+    ]
+
+    # Labels we determined we don't use
+    delete_labels = [
+        'duplicate',
+        'help wanted',
+        'invalid',
+        'wontfix',
+        'PR: ready for review',
+        'Status: duplicate',
+        'Status: invalid',
+        'Status: rejected',
+    ]
+
+    api = Github(os.environ.get('GITHUB_TOKEN'))
+    repo = api.get_repo(repo)
+
+    # First pass: create expected labels, try to repurpose old labels if
+    # possible first.
+    try:
+        existing_labels = list(repo.get_labels())
+    except GithubException:
+        print('Repository not found, check that `repo` argument is correct')
+        return False
+    for label in labels:
+        found = None
+
+        for existing_label in existing_labels:
+            if label.name == existing_label.name:
+                found = existing_label
+                break
+        if not found:
+            for existing_label in existing_labels:
```

Comment:
I think you can simplify this by doing `existing_label.name in ([label.name] + label.transpose)` instead of having two loops.

Reply:
The reason it is this way is to provide precedence to existing labels before it tries any transposed labels